tools/src/test/resources/io/github/ygojson/tools/dataprovider/impl/yugipedia/mapper/CardTable2MapperTest.testWikitextToCardTable2.continuous_spell.approved.txt

```diff
@@ -0,0 +1,49 @@
+CardTable2[cardclass=null, name=null, fr_name=Fissure Dimensionnelle, de_name=Dimensionsriss, it_name=Spaccatura Dimensionale, es_name=Fisura Dimensional, ja_name={{Ruby|次|じ}}{{Ruby|元|げん}}の{{Ruby|裂|さ}}け{{Ruby|目|め}}, romaji_name=Jigen no Sakeme, trans_name=null, base_romaji_name=null, base_trans_name=null, zh_name=null, zh_pinyin_name=null, zh_trans_name=null, ko_name=차원의 틈, ko_rr_name=null, ko_trans_name=null, alt_name=null, alt_name2=null, alt_name3=null, ja_alt_name=null, ja_alt_romaji=null, ja_alt_trans=null, image=DimensionalFissure-BLMR-EN-UR-1E.png, image2=null, card_type=Spell, property=Continuous, attribute=null, character=null, types=null, level=null, rank=null, link_arrows=null, pendulum_scale=null, maximum_atk=null, atk=null, def=null, password=81674782, limitation_text=null, ritualcard=null, ritualmonster=null, summon=null, effect_types=Continuous-like, pendulum_effect_types=null, ocg_status=null, tcg_status=null, tcg_speed_duel_status=null, source_card=null, skill_activation=null, fr_skill_activation=null, de_skill_activation=null, it_skill_activation=null, pt_skill_activation=null, es_skill_activation=null, pendulum_effect=null, fr_pendulum_effect=null, de_pendulum_effect=null, it_pendulum_effect=null, pt_pendulum_effect=null, es_pendulum_effect=null, ja_pendulum_effect=null, trans_pendulum_effect=null, zh_pendulum_effect=null, ko_pendulum_effect=null, lore=Any [[Monster Card|monster]] [[sent]] to the [[GY]] is [[banish]]ed instead., fr_lore=Les monstres envoyés au Cimetière sont bannis à la place., de_lore=Jedes Monster, das auf den Friedhof gelegt wird, wird stattdessen verbannt., it_lore=Qualsiasi mostro mandato al Cimitero viene invece bandito., pt_lore=Qualquer monstro enviado para o Cemitério, em vez disso, é banido., es_lore=Cualquier monstruo mandado al Cementerio es desterrado en su lugar., ja_lore=①：このカードが魔法＆罠ゾーンに存在する限り、お互いの墓地へ送られるモンスターは墓地へは行かず除外される。, trans_lore=null, zh_lore=null, ko_lore=①: 이 카드가 마법 & 함정 존에 존재하는 한, 서로의 묘지로 보내지는 몬스터는 묘지로는 가지 않고 제외된다., use=null, database_id=6707, en_sets=
```

Comment:
This output is polluted with a lot of null values and the multiple lines are only separating specific fields. I would rather test as a json, also like that we can document the mismatch on the java model with json-annotations and we can even have our first tool to output the yugipedia-cardtable2-model instead of directly ours. This tool might not be useful for our project (only for debugging purposes), but it might be useful for the community if they want to have the raw yugipedia data.

Reply:
Now we are using json for the tests. Also an issue is created to implement the tool: https://github.com/ygojson/ygojson-tools/issues/37